internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,17 +1,14 @@
 FROM hashicorp/terraform:light as terraform
 
-FROM python:3-alpine
+FROM ubuntu:20.04
 
-# required by gcloud SDK
-RUN apk add --no-cache git openssh curl
+RUN apt-get -q update && apt-get install -yq curl apt-transport-https ca-certificates gnupg && apt-get clean
 
-ENV GCLOUD_SDK_VERSION 369.0.0
-ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
-RUN curl "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_SDK_VERSION-linux-x86_64.tar.gz" > /tmp/google-cloud-sdk.tar.gz \
-  && mkdir -p /usr/local/gcloud \
-  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-  && /usr/local/gcloud/google-cloud-sdk/install.sh -q --override-components="bq" \
-  && rm /tmp/google-cloud-sdk.tar.gz
+ENV GCLOUD_SDK_VERSION 370.0.0-0
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
```

Comment:
I think you mixed all commands a bit.

1. Install GPG key and cloud-sdk repo
2. apt-get update -yq && apt-get install -yq curl apt-transport-https.... - google-cloud-sdk=${GCLOUD_SDK_VERSION} &&  apt-get clean

Reply:
`curl apt-transport-https gnupg` are required to add the deb repo and install the repo key, so two steps are required unfortunately. 
Would you prefer to split GPG key and repo addition from the actual package installation?

Reply:
You can try something like this:

```
apt-get update -yq && \ 
apt-get install apt-transport-https gnupg && \
deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
apt-get update -yq && \
apt-get install -yq google-cloud-sdk=${GCLOUD_SDK_VERSION} && \
apt-get clean
```

The aim is to consider this as an atomic piece to prevent any APT repo update from breaking the container build.

Reply:
I updated it with the suggestion. Just consider that if in the future another repo will be required I think would be best to extract the dependency installation so there is no implicit dependency.

Reply:
@mtojek I extracted it again, as Hashicorp has it's own repo that has the same requirements. It happened sooner than expected :D